builder_data.yaml

```diff
@@ -118,15 +116,8 @@ foss_platforms:
   - el-7-ppc64le
   - el-7-aarch64
   - eos-4-i386
-  - fedora-f25-i386
-  - fedora-f25-x86_64
-  - fedora-f26-x86_64
-  - fedora-25-i386
-  - fedora-25-x86_64
-  - fedora-26-x86_64
```

Comment:
Looks like agent (5.5.x) and pdk are still shipping fedora 26

Reply:
I know they're ripping fedora 26 out for this upcoming agent release. not sure about pdk..